test/hw2/test_task2.py

```diff
@@ -0,0 +1,23 @@
+from All_home_works.hw2.task2 import major_and_minor_elem
+
+
+def test_major_and_minor_elem():
+    input1 = [3, 2, 3]
+    output1 = 3, 2
+    assert output1 == major_and_minor_elem(input1)
+
+    input2 = [2, 2, 1, 1, 1, 2, 2]
+    output2 = 2, 1
+    assert output2 == major_and_minor_elem(input2)
+
+
+def test_major_and_minor_elem_non_most_common():
+    data = [1, 1, 1, 2, 2, 2, 0, 0, 0, 3]
+    output = (None, 3)
```

Comment:
Is this case mentioned in the requirements?

Reply:
In description: "The most common element is the element that appears more than n // 2 times.
The least common element is the element that appears fewer than others."
I decided to check it.

Reply:
Is anything else is related to it?

Reply:
There is nothing else related to it. Are you trying to say to me that I did unnecessary stuff?
But why they saying this: "more than n // 2 times." From this, I understand that in some data there is no most common element, for example: [1,2,3,4,5,6].

Reply:
> Are you trying to say to me that I did unnecessary stuff?

Yes. This is exactly what I am doing.  I try to make you notice some little details in the requirements, namely that phrase: "You may assume that the array is non-empty and the most common element
always exist in the array."

Reply:
Sorry! Haven't notice.